{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -182,16 +182,89 @@ class MediaRootGoogleCloudStorage(GoogleCloudStorage):
 # Django Admin URL regex.
 ADMIN_URL = env("DJANGO_ADMIN_URL")
 
-# Anymail (Mailgun)
+# Anymail
 # ------------------------------------------------------------------------------
 # https://anymail.readthedocs.io/en/stable/installation/#installing-anymail
 INSTALLED_APPS += ["anymail"]  # noqa F405
+{%- if cookiecutter.mail_service == 'Amazon SES' %}
+EMAIL_BACKEND = "anymail.backends.amazon_ses.EmailBackend"
+{%- elif cookiecutter.mail_service == 'Mailgun' %}
 EMAIL_BACKEND = "anymail.backends.mailgun.EmailBackend"
+{%- elif cookiecutter.mail_service == 'Mailjet' %}
+EMAIL_BACKEND = "anymail.backends.mailjet.EmailBackend"
+{%- elif cookiecutter.mail_service == 'Mandrill' %}
+EMAIL_BACKEND = "anymail.backends.mandrill.EmailBackend"
+{%- elif cookiecutter.mail_service == 'Postmark' %}
+EMAIL_BACKEND = "anymail.backends.postmark.EmailBackend"
+{%- elif cookiecutter.mail_service == 'Sendgrid' %}
+EMAIL_BACKEND = "anymail.backends.sendgrid.EmailBackend"
+{%- elif cookiecutter.mail_service == 'SendinBlue' %}
+EMAIL_BACKEND = "anymail.backends.sendinblue.EmailBackend"
+{%- elif cookiecutter.mail_service == 'SparkPost' %}
+EMAIL_BACKEND = "anymail.backends.sparkpost.EmailBackend"
+{%- elif cookiecutter.mail_service == 'Plain/Vanilla Django-Anymail' %}
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+# https://docs.djangoproject.com/en/3.0/ref/settings/#email-backend
+{%- endif %}
 # https://anymail.readthedocs.io/en/stable/installation/#anymail-settings-reference
 ANYMAIL = {
+    # https://anymail.readthedocs.io/en/stable/esps/
+    {%- if cookiecutter.mail_service == 'Amazon SES' %}
```

Comment:
It would _probably_ be more readable for us to limit the amount of `elif cookiecutter.mail_service ==` statements and group `EMAIL_BACKEND` and `ANYMAIL` dict in the same block so that they are next to each other, for instance:

```
{%- if cookiecutter.mail_service == 'Mailgun' %}
# https://anymail.readthedocs.io/en/stable/esps/mailgun/
EMAIL_BACKEND = "anymail.backends.mailgun.EmailBackend"
ANYMAIL = {
    "MAILGUN_API_KEY": env("MAILGUN_API_KEY"),
    "MAILGUN_SENDER_DOMAIN": env("MAILGUN_DOMAIN"),
    "MAILGUN_API_URL": env("MAILGUN_API_URL", default="https://api.mailgun.net/v3"),
}
...
```

That means we'll have to reapeat a bit the `ANYMAIL = {` part, but it will make it easier to follow all the settings for 1 provider. Right now it takes some overhead to think about the output for each provider.